_config.yml

```diff
@@ -0,0 +1,21 @@
+# Welcome to Jekyll!
+#
+# This config file is meant for settings that affect your whole blog, values
+# which you are expected to set up once and rarely need to edit after that.
+# For technical reasons, this file is *NOT* reloaded automatically when you use
+# 'jekyll serve'. If you change this file, please restart the server process.
+
+# Site settings
+title: Your awesome title
```

Comment:
Can we add `source: _source` or something so the top-level directory containing things like .gitignores, etc., don't get mixed into the jekyll tree?

I was about to make that change for vets-website too.